test/integration/connectors/test_kafka.py

```diff
@@ -47,20 +57,27 @@ def docker_compose_ctx():
         yield ctx
 
 
-def wait_for_topic(topic: str, retries: int = 10, interval: int = 1):
-    admin_client = get_admin_client()
+def wait_for_topic(
+    topic: str,
+    retries: int = 10,
+    interval: int = 1,
+    exists: bool = True,
```

Comment:
What purpose does adding this as an input help with? Seems odd to pass this in compared to the previous approach.

Reply:
It makes it possible to use this function to wait for both topic existance, and not-existance. I used it in the new test.